fpdf/template.py

```diff
@@ -117,21 +117,17 @@ def __getitem__(self, name):
                 # text for this page:
                 return self.texts[self.pg_no][key]
             # find first element for default text:
-            elements = [
-                element for element in self.elements if element["name"].lower() == key
-            ]
-            if elements:
-                return elements[0]["text"]
+            return next((x for x in self.elements if x["name"].lower() == key), None)
```

Comment:
Where did the `["text"]` went??

Reply:
Lost in translation. Sorry, thanks for the hint. Fixed.